### DISCUSSION TITLE
深圳封城记录 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#16] [Announcements]
# 深圳封城记录 - forecho's Blog

引言
受到疫情影响深圳于 2022 年 3 月 12 日发布了封城一周的政策，此篇文章主要记录我亲历的深圳封城一周的事情。
经历
关店
那天是 3 月 12 号周六，天气晴朗。昨晚我还跟电动车老板说电池不耐用的问题，他说我我平常用的小绿人充电桩充不满，叫我有时间拿去店里充。早上我把电动车拿去店里充电了，然后还在纠结要不要去健身。

https://blog.forecho.com/shenzhen-city-lockdown-record.html

**Comment by biabiabiu** [comment]
比上海强一些🤪

**Comment by hutusi** [comment]
身在上海，封在家二十多天了......

**Reply by forecho** [reply]
祝你们早日解封

**Comment by sunmking** [comment]
身在上海，封在家九十多天了......

**Reply by forecho** [reply]
现在还没解封？

**Reply by sunmking** [reply]
解封了
